Policies/github-community-guidelines.md

```diff
@@ -95,6 +104,10 @@ Actions we may take in response to an abuse report include but are not limited t
 * Account Suspension
 * Account Termination
 
+### Appeal and Reinstatement
+
+In some cases there may be a basis to reverse an action, for example, based on additional information a user provided, or where a user has addressed the violation and agreed to abide by our Acceptable Use Policies moving forward. If you wish to appeal an enforcement action, please contact [support](https://support.github.com/contact).
```

Comment:
I'd like to see the security community as part of the governance that decides appeals rather than Github support. Attitudes will change over time and certain situations will be thornier than others. If the security community is at the table (not the sole decision makers), the outcomes will be all the stronger for it.